Index: Window1.xaml
===================================================================
--- Window1.xaml	(revision 242424)
+++ Window1.xaml	(working copy)
@@ -2,16 +2,19 @@
 <syncfusion:ChromelessWindow x:Class="WizardSample.Window1" UseNativeChrome="True"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" Style="{StaticResource windowStyle}"
-    xmlns:syncfusion="http://schemas.syncfusion.com/wpf" syncfusion:SkinStorage.VisualStyle="{Binding SelectedItem,ElementName=Lstskin}" Icon="App.ico" WindowStartupLocation="CenterScreen"
+    xmlns:syncfusion="http://schemas.syncfusion.com/wpf" Icon="App.ico" WindowStartupLocation="CenterScreen"
         xmlns:local="clr-namespace:WizardSample"        
-    Title="Wizard Control Demo" Height="510" Width="750" Name="window1" ResizeMode="NoResize">
+    Title="Wizard Control Demo" Height="572" Width="774" Name="window1" ResizeMode="NoResize">
     <syncfusion:ChromelessWindow.DataContext>
         <local:ViewModel/>
     </syncfusion:ChromelessWindow.DataContext>
-	
-	<!-- Root Grid -->
+    <syncfusion:SkinStorage.VisualStyle>
+        <Binding Path="SelectedTheme"/>
+    </syncfusion:SkinStorage.VisualStyle>
+        <!-- Root Grid -->
 	<Grid>
 	<!-- Wizard Control -->
+        
 		<syncfusion:WizardControl Name="wizardControl" SelectedWizardPage="{Binding ElementName=selPages, Path=SelectedItem, Mode=TwoWay}"                                      
                                        HelpText="{Binding ElementName=txtHelpText, Path=Text, Mode=TwoWay}"
                                        CancelText="{Binding ElementName=txtCancelText, Path=Text, Mode=TwoWay}"
@@ -117,7 +120,7 @@
 			<syncfusion:WizardPage Name="wizPage7" Title="Skin" FinishVisible="False"
                                        Description="Select the skin"  BannerImage="/Image/W_O-BG.png" PageType="Exterior">
 				<GroupBox Header="Skin selection" Margin="25,20,25,5">					
-                    <ListBox ItemsSource="{Binding SkinCollection}" Name="Lstskin" SelectedIndex="10"
+                    <ListBox ItemsSource="{Binding SkinCollection}" Name="Lstskin" SelectedIndex="10" SelectedItem="{Binding SelectedTheme}"
                              Margin="5,3,5,3" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" 
                              BorderBrush="Transparent">
                         <ListBox.ItemTemplate>
