Index: ViewModel.cs
===================================================================
--- ViewModel.cs	(revision 242424)
+++ ViewModel.cs	(working copy)
@@ -116,6 +116,19 @@
             }
         }
 
+        private string  selectedTheme="Metro";
+        public string SelectedTheme
+        {
+            get { return selectedTheme; }
+            set
+            {
+                selectedTheme = value;
+                this.RaisePropertyChanged(() => this.SelectedTheme);
+            }
+        }
+
+       
+
         #endregion
 
         #region Constructor
